### ISSUE TITLE
Can not connect to a GPU other than a T4 for the past day on Pro+ account.

### ISSUE THREAD

**Issue by M3ATLOAF** [googlecolab/colabtools#3126]
The subject line pretty much explains it all.  It doesn't matter what time I try to connect, 
All I'm getting are T4s on a Pro+ account and that is not good considering the price I am paying or this service.

Please advise,
thanks

**Comment by cperry-goog** [comment]
You're selecting premium accelerator in runtime settings right?

**Comment by M3ATLOAF** [comment]
You mean here? 
![for github](https://user-images.githubusercontent.com/115184279/194398594-1edfa9e7-66a6-40ee-951d-e0c2ca0dee46.jpg)

**Comment by cperry-goog** [comment]
Check your compute unit balance - if you've run out then premium not accessible. Open a notebook, click on the arrow dropdown next to connect, and click on View resources.

**Comment by M3ATLOAF** [comment]
This is what I get. 
![image](https://user-images.githubusercontent.com/115184279/194402692-e844e2dd-b59a-4eec-a9b2-4da28aaafc22.png)

**Comment by M3ATLOAF** [comment]
I don't see anything that says "compute unit balance"

**Comment by M3ATLOAF** [comment]
Just to reiterate, I am not having problems connecting to a GPU since  a T4 is basically a GPU.  The problem is that I can not access anything higher than a T4 and a T4 or P100 are not fast enough for what I need to work on.  I've been running Pro+ for months now just fine, and this is something that just started happening.

**Comment by cperry-goog** [comment]
Looks like you've run out of quota. We recently made a change that allows you to utilize more Colab quickly as announced [here](https://blog.tensorflow.org/2022/09/colabs-pay-as-you-go-offers-more-access-to-powerful-nvidia-compute-for-machine-learning.html)

Your option is wait until your next recurrence or [purchase more compute units](https://colab.research.google.com/signup). Closing as this doesn't seem to be a bug.
